source/plugin/wgen-index.rst

```diff
@@ -0,0 +1,70 @@
+creation of the base terrain shape, the generation of biomes, and larger features such as villages and caves.
+
+.. Note::
+	Block placement in the generation phase takes place entirely within a buffer,
+	as the Chunk object is not created until the last step in the phase.
+
+In the beginning, there is nothing, and then the base generator populator creates the basic shape of the terrain.
+At this stage the world is entirely made of stone. Default generation is simply a heightmap generated from 
+perlin noise, then filled in - leaving air if higher than sea level, and water if below. This step can be overridden
+by calling ``setBaseGeneratorPopulator`` to provide your own custom populator to the world generator.
+
+Next, the biome generator populates a 2d biome buffer with the biomes that will correspond to each 1x1 column
+within the chunk. Using this biome buffer, the generator replaces the top layers of the terrain with the
+``BlockStates`` and depths specified by the biome's ``GroundCoverLayers``. The biomes are provided by a
+``BiomeGenerator`` - which may also be applied to the world generator with the ``setBiomeGenerator`` method. 
+
+Now that the base terrain has been generated and primed, we run through the ``GeneratorPopulator``\ s (*genpop*
+hereafter). These come in two groups: those specified globally for the world generator, and those specified for
+a specific biome. The set of genpops used is the union of the set of all genpops specified, for all biomes 
+present in the chunk.
+
+``GeneratorPopulator``\ s are designed for large intensive operations, and are applied to the buffer rather than
+the (as yet non-existent) Chunk object. This means that there must be the restriction that genpops may not place
+any blocks outside of the confines of the currently generating chunk. Content being generated by a genpop which
+spans multiple chunks must generate in pieces using the chunk position and world seed to determine which piece
+of the whole goes into the current chunk.
+
+Finally, with the base terrain created, the biomes applied, and the ``GeneratorPopulator``\ s run, we have
+completed the generation phase, and the Chunk object can be constructed and returned.
+
+The Population Phase
+--------------------
+
+The *Population Phase* can only run on a chunk once the three chunks adjacent to it in the positive x and z axes
+are loaded. This allows objects during this phase to expand outside the 16x16 chunk area being populated, without
+requiring the partial generation used for ``GeneratorPopulator``\ s. To support this, the actual area populated
+during this phase is a 16x16 area offset by 8 in each of the x and z axes.
+
+[Image TBA ?]
+
+At this time the populators defined globally and those defined for the biome specified for the centre block of
+the populating region (which is the maximal block of the chunk passed to the populator due to the offset
+specified above). ???
```

Comment:
What I was trying to saw was that only the populators of the biome at the position `(x*16+16, 0, z*16+16)` are applied to this area. Not a union of all the biomes as is the case for  GenerationPopulators